.github/workflows/pdf-build.yml

```
@@ -27,18 +28,20 @@ jobs:
             ${{ runner.os }}-node-
       - name: Install marp
         run: |
-          npm install --save-dev @marp-team/marp-cli
+          npm install --save-dev @marp-team/marp-cli markdown-pdf
       - name: Build PDFs
         run: |
-          for input in `grep -l 'marp: true' *.md */*.md`
-          do
-            output=`echo -n $input | sed -e 's/.md$//g'`
-            echo "$output.pdf"
-            npx marp --allow-local-files "$input" -o "$output.pdf"
+          for input in *.md */*.md; do
+            output=`echo -n $input | sed -e 's/\.md$/.pdf/g'`
+            echo "Input: $input, output $output";
+            if [[ $(grep -l 'marp: true' $input) ]];
+            then
```

```suggestion
            if [[ "$(grep -l 'marp: true' "$input")" ]]; then
```

Same as `for; do` and also ensure files with spaces in the name don't cause issues by quoting `$input`. Further, it's best practice to quote the output of scripts without a good reason to not do it